### PR TITLE
Installation instructions

### PR DESCRIPTION
**Pull request by dpshelio**
Adds principally the installation instructions through the docs.
It also adds some changes on how the setup files are.
Removes some redundant code on travis and uses tox-conda to include it all there.

### CODE REVIEW

File: docs/installation.rst [hunk 1]
```diff
@@ -4,10 +4,41 @@
...
+We suggest you use `anaconda`_ (or `miniconda`_) with Python 3.7 for the
+installation. However, you can also get the dependencies through your prefered
+package manager.
+
+Using `conda` you can create an isolated environment for KaSKA.
+
+.. code-block:: console
+
+   $ conda config --add channels conda-forge
+   $ conda create -n kaska python=3.7 tensorflow numba gdal
```
Comment (dpshelio) [inline docs/installation.rst]: version of gdal

Comment (dpshelio) [reply docs/installation.rst]: Tensorflow maybe smaller than 2.

Comment (mattgillucl) [reply docs/installation.rst]: Presumably, as per issue #87, we need to set maximum version of `goal`?
i.e. `gdal"<=2.5.2"`

**Comment by mattgillucl** [comment]
Presumably, as per issue #87, we need to set maximum version of `gdal`?
i.e. `gdal"<=2.5.2"`

File: docs/installation.rst (hunk 1 above)
Comment (timspainUCL) [reply docs/installation.rst]: See PR #91

**Comment by dpshelio** [comment]
All :green_heart: now. Anyone can approve it? :)

**Pull request merged**